test/data/fra-prepared.xml

```diff
@@ -11,7 +11,7 @@
                     <s>Et voici une deuxième ligne dans le même paragraphe.</s>
                 </p>
                 <p>
-                    <s>Un deuxième paragraphe précédé d&#x27;une ligne vide est ici. Avec deux phrases.</s>
+                    <s>Un deuxième paragraphe précédé d'une ligne vide est ici. Avec deux phrases.</s>
```

Comment:
It took me a while to figure out why you made this change, since it's semantically the same, but I see chevron outputs literal `'` instead of `&#x27;`. It's much nicer this way, so good.